utility/mm2_update-master-directory-list

```diff
@@ -138,6 +158,7 @@ def make_file_details_from_checksums(session, diskpath, relativeDName, D):
                 size=size)
             session.add(fd)
             session.commit()
+            logger.debug("Added checksum for %s to database" % f)
```

Comment:
I _believe_ if you pass ``f`` as an argument rather than substituting it - ``logger.debug("Added checksum for %s to database", f)`` - it will cause the substitution to occur when logging at the debug level is on.

That's probably not going to be a huge performance gain in the grand scheme of things, but it might add up. Just an FYI.